apps/condo/domains/banking/schema/BankAccount.js

```diff
@@ -230,6 +230,38 @@ const BankAccount = new GQLListSchema('BankAccount', {
                 },
             },
         },
+
+        name: {
+            schemaDoc: 'Bank account name to display',
+            type: 'Text',
+            isRequired: false,
+            hooks: {
+                resolveInput: async ({ operation, fieldPath, resolvedData, existingItem }) => {
```

Comment:
you do not need this hook if you will choose what to display on the client
It will be good to use financeInfoClient to set default name